packages/ui/src/components/Accordion/Accordion.stories.tsx

```diff
@@ -1,16 +1,14 @@
 import React from 'react'
 
-import { Accordion } from '.'
-import { Typography } from '../Typography'
-import { IconArrowUp } from '../Icon/icons/IconArrowUp'
+import Accordion, { AccordionProps } from './Accordion'
 import { Badge } from '../Badge'
 
 export default {
   title: 'Displays/Accordion',
   component: Accordion,
 }
 
-export const OneItem = (args: any) => (
```

Comment:
see other comment, pls refactor to kitchen sink style

Reply:
is there a reason for kitchen sink ?

Reply:
whats the outcome you expect so everyone is aligned ?
kitchen sink might mean something different to others as well

Reply:
leaving for now